tools/c7n_gcp/c7n_gcp/actions/notify.py

```diff
@@ -0,0 +1,153 @@
+# Copyright 2018 Capital One Services, LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from googleapiclient.errors import HttpError
+from c7n.actions import BaseNotify
+from c7n import utils
+from c7n.resolver import ValuesFrom
+from c7n_gcp.provider import resources as gcp_resources
+
+
+class Notify(BaseNotify):
+    """Example:
+        - name: gcp-notify-with-attributes
+          resource: gcp-compute
+          filters:
+           - Name: bad-instance
+          actions:
+           - type: notify
+             to:
+              - event-user
+              - resource-creator
+              - email@address
+             owner_absent_contact:
+              - other_email@address
+             # which template for the email should we use
+             template: policy-template
+             transport:
+               type: pubsub
+               topic: your-notify-topic
+    """
+    batch_size = 250
```

Comment:
batch_size can be significantly large on gcp as the message limits on pub/sub are significantly higher then other transports.

Reply:
Kept it small on purpose during testing. Latest commit will bump it up higher.

Reply:
bumped to 1k